layouts/shortcodes/include-markdown.html

```diff
@@ -3,9 +3,11 @@
 {{- with .Site.GetPage $page -}}
     {{ .Content }}
 {{- else -}}
-    {{- if eq (os.Getenv "LOCAL_TRANSLATION_BUILD") "true" -}}
-        {{ warnf "[include-markdown shortcode] Page %s not found in %s (language: %s)" $page .Page.File.Path .Page.Lang }}
-    {{- else -}}
-        {{ errorf "[include-markdown shortcode] Page %s not found in %s (language: %s)" $page .Page.File.Path .Page.Lang }}
-    {{- end -}}
+    {{ warnf "[include-markdown shortcode] Page %s not found in %s (language: %s)" $page .Page.File.Path .Page.Lang }}
+    {{ with where .Site.Sites "Language.Lang" "eq" "en" }}
+        {{/*  use english markdown for Content as default  */}}
+        {{ $enSite := index . 0 }}
```

Comment:
I think you might just be able to use `.Sites.Default` or `.Sites.First` to reduce some of this logic

Reply:
i updated it to use First instead